infrastrucutre/certificate.tf

```diff
@@ -14,3 +14,20 @@ module "acm" {
 
   wait_for_validation = true
 }
+
+data "aws_lb" "app_lb" {
```

Comment:
Keep in mind that this is chicken and the egg problem as first you need to create ingress in order to be able to get the Load Balancer info.

Also you cannot search for it via static variable but you can use tags instead or even better maybe to utilise kubernetes data resource to find out exact hostname of the ingress that was created.